float64[] x
float64[] y
float64[] z
float64[] yaw

float64 totalTime
bool is_timeScaled
float64[] u

bool is_timeInverted
bool is_loopPiece
bool is_yawFollowing

string frame
